evergreen.yaml

```diff
@@ -79,7 +79,7 @@ tasks:
     name: race
 
   - <<: *run-build
-    tags: ["test"]
+    tags: ["bench"]
```

Comment:
I don't see any variants that run this tag.

Reply:
I added tasks to the evergreen.yaml.